### DISCUSSION TITLE
Hiding Meshcentral from detection

### DISCUSSION THREAD

**Discussion by zakwan05** [Ylianst/MeshCentral#6003] [Q&A]
Hi there Meshcentral community!!
I wanted to ask if anybody here can help me with hiding meshcentral from detection by any third-party service. Right now its being detect by Pearson Vue and kinda defeats the entire purpose of hidden remote access..
Help would be appericiated. Any suggested change in config or mesh in general?

**Comment by si458** [comment]
NO,
the meshagent doesnt want to be hidden as it is remote support software,
so it needs to be shown/visible to let people know they are being remotely controlled.

**Comment by silversword411** [comment]
Don't cheat on tests.
People that cheat on tests should be expelled
